pkg/controller/pxc/users.go

```diff
@@ -131,13 +130,13 @@ func (r *ReconcilePerconaXtraDBCluster) manageMonitorUser(cr *api.PerconaXtraDBC
 		pxcPass = string(internalSysSecretObj.Data["operator"])
 	}
 
-	um, err := users.NewManager(cr.Name+"-pxc-unready."+cr.Namespace+":33062", pxcUser, pxcPass)
+	um, err := users.NewManager(cr.Name+"-pxc-unready."+cr.Namespace+":3306", pxcUser, pxcPass)
```

Comment:
@delgod  @nonemax  will it work with proxy protocol ? maybe we need to use 3306 for <= 1.5.0 and 33062 for >= 1.6.0

Reply:
we need this only in upgrade cases, so there is no 33062 port in point where this method run

Reply:
we run it only once during the early update to 1.6.0.
we have check above

Reply:
but we can detect if `cr.pxc.configuration` has proxy_protocol key and change port to 33062 in case its in it

Reply:
we need to use 33062 port for all needs for >= 1.6.0 , it is admin/service port

Reply:
We cannot rely on version checking because all the magic with users happens at the beginning of the loop. Thus, having detected version 1.6.0, we will knock on port 33062, which is not yet open. So the only way for now is detect proxy_protocol and use 33062 only if its present